README.md

```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
 
-Many of the issues reported by the Security Audit have Fixes associated with them. These are code snippets that you can get inserted into the OpenAPI file and then customize with the appropriate value.
+- **A credential** : most likely, your API is using some form of authentication, like an API Key or token. You need a valid credential to provide to the scan engine. 
 
-#### Individual fixes
+- **The URL** when the API is deployed.  
 
-1. Perform the Security Audit by clicking the **42c** button.
-2. When you click some of the error lines in your OpenAPI file, you will see the QuickFix icon show up on the left of the line.
-3. Click the icon and select the QuickFix to apply.
-4. Tweak the values in the inserted code snippet as you see fit.
+  ![](https://img.shields.io/badge/Warning-orange)  We recommend that you do <u>not</u> target a production system. While the tool does not try to inject malicious payloads, it is possible that the API implementation is not resilient enough to handle the tests and may crash or behave unexpectedly.
 
-![Quick Fixes in VSCode](/images/Quick_Fixes_in_VSCode.gif)
+When you first launch a scan, you will be presented with the scan configuration viewer. The scan configuration is generated automatically from the OpenAPI file you chose to scan. 
 
-#### Bulk fixes
+- For each operation in the OpenAPI file, a request is created. You can test individual requests using the **Try** button top-right. 
 
-If your OpenAPI file contains multiple similar issues - for example, a parameter with the same name is missing the pattern definition - you can fix all such occurrences in one go:
+  ![](./images/ScantryIt.png)
 
-1. Fix the first occurrence and supply the proper values in the code snippet.
-2. Go to the next occurrence and use the bulk QuickFix. This bulk QuickFix will reuse the values that you used previously for the parameter with the same name.
+- Requests can be arranged into testing scenarios: a scenario combines one or multiple requests, for example you need to create a resource before you can view it. The editor allows you to extract data as variables from a request execution to inject it into the next step.
+	
+	Similarly to requests, you can test an individual scenario to ensure it is built correctly, before starting a scan: the scan engine will execute the scenario and if successful, launch automatically dozens of test, using the data provided in the OpenAPI file. It will send bad verbs, bad data types, bad data formats as well as authentication tests. 
+	
+	![](/Volumes/MYDATA/Product/vscode-openapi/images/ScanScenarios.png)
+	
 
-![Bulk Fixes in VSCode](/images/VSCode_OpenAPI_bulk_security_fixes.gif)
+![](https://img.shields.io/badge/Learning%20More!-green)  <link to video tutorial> 
 
 ## Miscellaneous commands
 
 - You can load results of Security Audit from a file for a currently open OpenAPI file by running command `42Crunch: Load Security Audit report from file` from the command palette
```

Comment:
Hmmm, not sure if this should be "Security Audit" or "42Crunch Audit". See which you think works better here, both could be justified (but for different reasons).